quarkchain/cluster/grpc_client.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import sys
+import grpc
+
+from quarkchain.generated import grpc_client_pb2_grpc
+from quarkchain.generated import grpc_client_pb2
+
+
+class GrpcClient(object):
+    def __init__(self, channel):
```

Comment:
the channel should be encapsulated inside the constructor, so the caller doesn't need to understand what it is